conf/tasks/Makefile.gke

```diff
@@ -275,3 +279,31 @@ gke/destroy/all: \
 	gke/destroy/pds
 	@echo "GKE cluster destroyed"
 	@exit 0
+
+
+
+# https://cloud.google.com/storage/docs/access-control/iam-roles
+# Currently, using Editor and Kubernetes Engine Admin roles for the testing-ci service account.
+# It might be possible to replace Editor with someting more specific, but more research would be needed.
+## Create Service Account used by deepcell
+gke/test/create/service-account:
+	@echo "Creating GKE service account..."
+	@gcloud auth activate-service-account $(GCP_SERVICE_ACCOUNT) --key-file=$(CONF_PATH_PREFIX)/gcloud_key.json
+	@echo "GKE service account creation complete."
+	@echo " "
+	@echo " "
+
+## Create Cluster
+gke/test/create/all: \
+	gke/test/create/service-account \
+	gke/create/resources
+	@echo "GKE cluster created"
+	@exit 0
+
+## Destroy Cluster
+gke/test/destroy/all: \
+	gke/destroy/node-pools \
+	gke/destroy/cluster \
+	gke/destroy/pds
+	@echo "GKE cluster destroyed"
+	@exit 0
```

Comment:
If this fails, how can we clean up the resources that didn't get destroyed?

Reply:
Great point. I opened #274 off of this comment. This seems to be a problem in production, too.